sources/packages/web/src/services/ApplicationService.ts

```diff
@@ -3,7 +3,7 @@ import {
   SaveStudentApplicationDto,
   ProgramYearOfApplicationDto,
   ApplicationStatusToBeUpdatedDto,
-  GetApplicationDataDto,
+  GetApplicationDataDto
```

Comment:
It seems weird to me that this file has "comma removal" as the only change, and the next file right below has the comma. The formatter is supposed to add the comma automatically.
![image](https://user-images.githubusercontent.com/61259237/135678999-91a84836-bd00-4606-8a9a-14e5b948c549.png)